addons/base_iban/static/src/components/iban_widget/iban_widget.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="base_iban.iban" t-inherit="web.CharField" owl="1">
+        <xpath expr="//input" position="attributes">
+            <attribute name="t-on-input">onInputValidateIban</attribute>
```

Comment:
why not just call the debounced method directly?

Reply:
Because of a small improvement you told me that would be very nice to have.
When the user is typing, I don't want to call the validation method directly BUT I want to set the `isValidIBAN` to `null` to get rid of the validation icon.
This allows that behavior

Reply:
I prefer it just stays red until its correct and disappears - instead of appearing and disappearing.
what do you think?

Reply:
In terms of UX, I don't like it: If it stays red while the users is typing it could lead them to think that it is evaluated in real-time.

Not a big issue because it is just a 400ms delay after typing stopped and users won't enter their IBAN a lot but it's missleading.